plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -168,7 +168,9 @@
 public abstract class CitrixResourceBase implements ServerResource, HypervisorResource, VirtualRouterDeployer {
 
     public enum SRType {
-        EXT, FILE, ISCSI, ISO, LVM, LVMOHBA, LVMOISCSI, NFS;
+        // RELVMOISCSI = used for resigning metadata (like SR UUID and VDI UUID when a
+        // particular storage manager is installed on a XenServer host (for back-end snapshots to work))
+        EXT, FILE, ISCSI, ISO, LVM, LVMOHBA, LVMOISCSI, RELVMOISCSI, NFS;
```

Comment:
@mike-tutkowski Sorry if I am being too repetitive.

As an idea, this comments might serve well as a Javadoc block documenting the enum class. If you are friendly with this idea, the same could be done with the **org.apache.cloudstack.engine.subsystem.api.storage.DataStoreCapabilities** enum class.

Thanks.